lib/modules/platform/bitbucket/index.ts

```diff
@@ -949,3 +954,81 @@ export async function mergePr({
   }
   return true;
 }
+
+export async function getReleaseNotesMd(
```

Comment:
i don't like to have those functions here. they should stay inside changelogs code

Reply:
My longer-term thought process was:
- extend the platform function interface to include release-notes functions (raw rile, tags, release-notes etc)
- all platform/* types could implement these functions (or return non-implemented)
- avoids having http client logic for the same platform across two places

Reply:
I don't think you can reuse the platform http, because releases notes are not always on the same system where renovate runs against. This would currently only work for bitbucket cloud, as there is no self-hosted variant. but will not work for all other platforms.

So i prefer to leave that on changelog code.

@rarkins @secustor @JamieMagee  WDYT?

Reply:
In favor leaving as is currently, as we have been talking about a bigger changelog revamp on and off lately.

Reply:
ok then - moved back to the changelog code